packages/client/package.json

```diff
@@ -39,7 +39,6 @@
     "@slidev/types": "workspace:*",
     "@typescript/ata": "^0.9.4",
     "@unhead/vue": "^1.9.5",
-    "@unocss/reset": "^0.59.3",
```

Comment:
Removed these packages because they are not used by `@slidev/client` in fact.

Reply:
Even if the paths are resolved on the node side, the client package is, in fact, dependent on them. I think it's better to have them in client's package.json to be clear

Reply:
Oh, that's my mistake. I forgot they are actually providing some CSS to client.